check50.py

```diff
@@ -32,8 +33,8 @@ def main():
     parser.add_argument("files", nargs="*")
     parser.add_argument("-d", "--debug", action="store_true")
     parser.add_argument("-l", "--local", action="store_true")
-    parser.add_argument("-f", "--force", action="store_true")
     parser.add_argument("--log", action="store_true")
+    parser.add_argument("--no-autoupdate", action="store_true")
```

Comment:
I was just trying to account for the use case that a student just runs `check50` with no arguments and we automatically check for updates. But yeah, if we alias `check50` to `check50 --update`, that resolves the issue. Can flip this to `--update` if that works better for offline users (offline users, for what it's worth, will need `check50` aliased to `check50 --local`).